meson.build

```diff
@@ -260,10 +260,12 @@ cc = meson.get_compiler('c')
 pkgconfig = import('pkgconfig')
 check_compilation_sh = find_program('tools/meson-check-compilation.sh')
 
-cxx = find_program('c++', required : false)
-if cxx.found()
-        #  Used only for tests
-        add_languages('cpp')
+if get_option('tests') != 'false'
+        cxx = find_program('c++', required : false)
+	    if cxx.found()
```

Comment:
8 spaces indentation everywhere, and the closing statements must align with the opening ones.

Reply:
whoops! I have draw spaces on and missed it. Will push in a second.